### PR TITLE
First pull request for Hello Microverse project

### PR DESCRIPTION
**Pull request by TechnoAndy**
First, pull request after completing the requirements as listed in the exercise.

### CODE REVIEW

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @TechnoAndy 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on to set up your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Readme is documented
✔️  Correct PR title
✔️  Commit messages are meaningful

## Required Changes ♻️
 
 - [ ] You currently do not have any linter checks configured. The `linters.yml` file should have code found in the [linter for HTML and CSS template](https://github.com/microverseinc/linters-config). Kindly copy [this code](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) and paste it inside your `linters.yml` file to set up linter checks.
 - [ ] When you are done, make sure that  all linter checks pass
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
-# Hello_Microverse
-First project in the Full Stack Program
+Project Name
```
Comment (ichala) [inline README.md]: - [ ] [OPTIONAL] kindly add your project name in here to make your readme file more professional

File: .github/workflows/name.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (ichala) [inline .github/workflows/name.txt]: - [x] [OPTIONAL] kindly delete this file `(name.txt)` since its not useful and empty

**Review by ichala — APPROVED** [review]
Hi @TechnoAndy ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻
✅ You used the correct Github Flow.
✅ Professional README.
✅ Linters are working perfectly now and without errors.



Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- [ ] **_Check the comments bellow_**


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/name.txt (hunk 2 above)
Comment (TechnoAndy) [reply .github/workflows/name.txt]: The file has been removed. Thank you

**Pull request merged**